native/common/jp_encoding.cpp

```diff
@@ -217,37 +217,6 @@ unsigned int JPEncodingUTF8::fetch(std::istream& in) const
 	return -1;
 }
 
-//*****************************************************************
```

Comment:
Why did you delete this hilarious comment? I just tripped on it now after so many years - made my day

Reply:
My employeer was less than thrilled.    So sadly it must go away.

Reply:
Everywhere popular politics. How did they become aware of this? Are there legal concerns?

Reply:
I have been *ahem* slightly skirting the limits of my laboratory policies.  When I joined this line of work in 2001 I had an agreement that I could operate on open source so long as it was not related or had security concerns.  But since then we have changed  contracts and someone decided to publish a new policy that restricts my contribution to no more that "25%" of the total project code.  Of course no one pushed to policy to any group leaders so it apparently sat on some web site until I was pushing another open source project out the door, which me being a good citizen recognized as an issue, so I followed the procedure.   Lawyers were involved and way more time spent as work that I have ever spent on this project there (seeing as I only work on it at night and at 5 AM).   Needless to say I was given a list of items to immediately change, a trip to the ethics office to explain why doing stuff for free does not count as a conflict of interest, and after all that the lawyer concluded (paraphrasing) "no conflict, typical professional activity for a programmer, no concerns", but no one is willing to sign the paper to either release the code or claim the code.  Should they decide to claim the code a "code release number" will be assigned and the "auspices statement" will need to be added claiming this code was developed with gov't dollars, but thus far given that the review was the only thing that ever cost the gov't a penny noone appears willing to give me that stamp.  I make the call once a month (every time the automatic system sends me a notice) to all IP and ask them to complete the release paper but given noone will every make a cent on this project it is a very low priority.  Once "released" I am allowed to file with copyright office to assert that I own the code (not my employer) I may have to do just to make sure no one who gets our contract in the future can raise the claim again.  Then I will then stick it in a drawer to be found in my personal papers sometime in the year 2060.

Bottom line... I had to put it through review which required the comment and a handful of other off-color comments from other contributors to be removed.  Ethics office has agreed that I am operating in good faith contributing to software used by the lab.  IP office has concluded they will define "25%" as whatever number I have contributed and "Apache" as an acceptable license as a policy they arbitrarily defined is not a legal matter.  One tree died a pointless death. 

*LEGAL: This comment is a somewhat humorous restatement life at the laboratory, and does in no way impute the honor of the laboratory or the management.  Opinions expressed at 6 AM do not reflect the opinions of the laboratory nor the US government.*